src/templates/_components/fieldtypes/Matrix/input.twig

```diff
@@ -2,7 +2,7 @@
 
 <div class="matrix matrix-field" id="{{ id }}">
     <span aria-live="assertive" class="visually-hidden" data-status-message></span>
-    <div class="blocks">
+    <ol class="blocks">
```

Comment:
Potentially-breaking DOM changes like this should be postponed until the next major release (Craft 5).